eva_cttv_pipeline/trait_mapping/output.py

```diff
@@ -0,0 +1,66 @@
+import csv
```

Comment:
Could we have tests added for some of these functions? This file has a 0% test coverage.